layouts/_default/list.html

```diff
@@ -6,14 +6,14 @@
   {{ end }}
   <div class="posts">
```

Comment:
Hey, thank you for the PR and fixes 🙌 I wonder whether we should change this `div` to `article` as well?